app/controllers/inventory_foods_controller.rb

```diff
@@ -0,0 +1,30 @@
+class InventoryFoodsController < ApplicationController
+  def new
+    @new_inventory_food = InventoryFood.new
+  end
+
+  def create
+    inventory_food = InventoryFood.new(inventory_food_params)
+    respond_to do |format|
+      if inventory_food.save
+        flash[:notice] = 'Created an inventory food succesfully'
+        format.html { redirect_to "/inventories/#{params[:id]}" }
+      else
+        flash[:notice] = 'Failed to create an inventory food. Try again'
+        format.html { redirect_to "/inventories/#{params[:id]}/inventory_foods/new" }
+      end
+    end
+  end
+
+  def destroy
+    inventory_food = InventoryFood.find(params[:id])
+    inventory = inventory_food.inventory
+    inventory_food.destroy
+    flash[:notice] = 'Inventory food was successfully removed'
+    redirect_to "/inventories/#{inventory.id}"
+  end
+
+  def inventory_food_params
+    params.require(:inventory_food).permit(:inventory_id, :food_id, :quantity)
+  end
```

Comment:
Hey @akucintavalent , 

- [ ] It's good to make the ` inventory_food_params` method a private. That's considered a good practice.